bills-app/src/main/webapp/script.js

```diff
@@ -12,39 +12,794 @@
+                "latest_major_action": "Referred to the House Committee on Natural Resources."
+            },
+            {
+                "bill_id": "hr4242-115",
+                "bill_slug": "hr4242",
+                "bill_type": "hr",
+                "number": "H.R.4242",
+                "bill_uri": "https://api.propublica.org/congress/v1/115/bills/hr4242.json",
+                "title": "To amend title 38, United States Code, to establish a permanent VA Care in the Community Program, and for other purposes.",
+                "short_title": "To amend title 38, United States Code, to establish a permanent VA Care in the Community Program, and for other purposes.",
+                "sponsor_title": "Rep.",
+                "sponsor_id": "R000582",
+                "sponsor_name": "Phil Roe",
+                "sponsor_state": "TN",
+                "sponsor_party": "R",
+                "sponsor_uri": "https://api.propublica.org/congress/v1/members/R000582.json",
+                "gpo_pdf_uri": null,
+                "congressdotgov_url": "https://www.congress.gov/bill/115th-congress/house-bill/4242",
+                "govtrack_url": "https://www.govtrack.us/congress/bills/115/hr4242",
+                "introduced_date": "2017-11-03",
+                "active": false,
+                "last_vote": null,
+                "house_passage": null,
+                "senate_passage": null,
+                "enacted": null,
+                "vetoed": null,
+                "cosponsors": 23,
+                "cosponsors_by_party": {
+                    "R": 13,
+                    "D": 10
+                },
+                "committees": "House Veterans&#39; Affairs Committee",
+                "committee_codes": ["HSVR"],
+                "subcommittee_codes": [],
+                "primary_subject": "",
+                "summary": "",
+                "summary_short": "",
+                "latest_major_action_date": "2017-11-03",
+                "latest_major_action": "Referred to the House Committee on Veterans' Affairs."
+            }
+        ]
+    }]
 }
 
-function addCode() {
-    getUpcomingBillsData();
+// Fetched the upcoming bills from the api
+const getUpcomingBills = () => {
+    // do api logic here
+
+    renderBills(mockResponse.results[0].bills);
 }
 
-function getUpcomingBillsData() {
-    //api key is declared in api.js as: 'proPublicaApiKey'
-    const url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
-    fetch(url, {
-        method: "GET", 
-        headers: {
-            "X-API-Key": proPublicaApiKey
-        }
-    }).then(response => response.json()).then((upcomingBills) => {
-        //printing the json file to console
-        console.log(upcomingBills);
-
-        //Prints out the contents of "status" from house.json into the website. 
-        //If everything goes well, prints out "OK"
-        document.getElementById('API-status-container').innerText = upcomingBills.status;
```

Comment:
You definitely want to keep this code and not delete it :) This just needs to be moved to where you wrote `//do api logic here` above, and in the callback instead of logging the bills, you call your renderBills().

Reply:
I just realized this would be getting ahead a bit into what you're supposed to be doing this week. Hopefully this is easy enough to do, or you can keep both code (fetching + rendering based on mock data) separate for this PR (so the bills data is fetched, but you don't do anything with the actual data yet).